changelog.d/gh-4722.added

```diff
@@ -0,0 +1,8 @@
+Added lone decorators as a valid Python semgrep pattern, so for example `$NAME($X)` will
```

Comment:
Typo here, it should have been `@$NAME($X)` I guess.

Reply:
Nice changelog entry BTW! Pretty easy to understand for anybody as it should be.